src/sql/workbench/contrib/commandLine/electron-browser/commandLine.ts

```diff
@@ -136,6 +137,9 @@ export class CommandLineWorkbenchContribution implements IWorkbenchContribution,
 				// and b) use the latest version of the profile from the service so most fields are filled in.
 				let updatedProfile = this._connectionManagementService.getConnectionProfileById(profile.id);
 				connectedContext = { connectionProfile: new ConnectionProfile(this._capabilitiesService, updatedProfile).toIConnectionProfile() };
+				if (args.showDashboard) {
+					this._connectionManagementService.showDashboard(updatedProfile);
```

Comment:
The error on 144 is going to be wrong if this throws

It looks like you can also just set the 'purpose' to dashboard if you want to show the dashboard in the original `connectIfNotConnected` call so maybe that'd be better. Although I guess it still has the same issue probably as this with the error message...but I do like using common functionality when possible